app/routes/user/email_notifications.server.ts

```diff
@@ -0,0 +1,154 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+import { tables } from '@architect/functions'
+import { getUser } from '~/routes/__auth/user.server'
+import {
+  mapFormatAndNoticeTypesToTopics,
+  mapTopicsToFormatAndNoticeTypes,
+} from '~/lib/utils'
+
+export type EmailNotification = {
+  name: string
+  recipient: string
+  format: string
+  noticeTypes: string[]
```

Comment:
I dont believe these would be known to the db, since they are not included in the put, but I have updated the file to have a view model version that includes these fields for the purpose of the eventual Edit page, and updated the methods to accept and return the appropriate models based on function